### DISCUSSION TITLE
Bug on modal close

### DISCUSSION THREAD

**Discussion by eincandela** [wire-elements/modal#277] [Q&A] [answered]
i get this console error when i close a modal

```
cdn.min.js:5 Uncaught (in promise) TypeError: u is not a function
    at cdn.min.js:5:9014
    at async Promise.all (index 3)

```    
After that, no modals open up anymore
  
(I can temporary fix this by going back to alpine.js v2 instead of v3)

**Comment by PhiloNL** [comment, marked as answer]
I can't reproduce this issue. Please verify you don't load Alpine twice, as this is a common cause of errors like this.

**Comment by eincandela** [comment]
You are right. Thanks a lot. Alpine was already included in livewire js :)
